.github/workflows/build.yaml

```diff
@@ -0,0 +1,62 @@
+name: Build Openbox, Recorder
```

Comment:
`nitpick`  could be "... Openbox and Recorder" instead of "... Openbox, Recorder"